Dockerfile

```diff
@@ -23,8 +24,16 @@ RUN git clone https://github.com/sqlcipher/sqlcipher && \
     make install && \
     ldconfig
 
+RUN python3 -m pip install --upgrade pip
 COPY ./requirements.txt /app/requirements.txt
+
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+ENV PATH="/root/.cargo/bin:${PATH}"
+RUN rustup default nightly-2021-03-24 
```

Comment:
I have zero experience with rust, but out of curiosity is this using a nightly build of the rust toolchain? and if so the is there any specific reason for this?

Reply:
Actually no, I didn't investigate this further after the project compiled. In the beginning, because of the aforementioned error, I thought that a feature in lock_api for some reason was made available only to nightly builds.